src/components/App.css

```diff
@@ -0,0 +1,318 @@
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.9rem;
+  letter-spacing: -0.4px;
+}
+
+.progress-bar h4 {
+  width: 2.938rem;
+  height: 2.438rem;
+  margin: 0.063rem 7.063rem 0 0.688rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  margin-top: 1rem;
+}
+
+.progress-bar p {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0.438rem 4.938rem 0 0.75rem;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+}
+
+.book-detail .desc h4 {
+  width: 7rem;
+  height: 1.125rem;
+  margin: 0 4rem 0 0;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.book-detail .desc h2 {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.progress-bar {
+  display: flex;
+  justify-content: space-between;
+  border-right: solid 1px #e8e8e8;
+}
+
+.book-detail .desc p {
+  width: 5.813rem;
+  height: 1.188rem;
+  margin: 0.25rem 0.875rem 0 0;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.868rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.book-detail .desc .links {
+  display: flex;
+  margin-top: 1.313rem;
+}
+
+.book-detail .desc .links p {
+  width: 4.563rem;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+}
+
+.book-detail .desc .links p button {
+  border: none;
+  background-color: white;
+  width: 3.375rem;
+  height: 1.188rem;
+  margin: -2px 0.938rem 0.125rem 0.875rem;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.book-detail .desc .links .edit {
+  margin: 0.188rem 18.625rem 0.125rem 1.063rem;
+}
+
+.book-detail .desc .links .comment::after {
+  content: '';
+  width: 0.125rem;
+  height: 1.5rem;
+  margin: 0.75rem 1.063rem 0 1.2rem;
+  border: solid 1px #e8e8e8;
+}
+
+.book-detail .desc .links p button::after {
+  content: '';
+  width: 0.125rem;
+  height: 1.5rem;
+  margin: 0.75rem 1.063rem 0 1.2rem;
+  border: solid 1px #e8e8e8;
+}
+
+#filter {
+  margin-left: 50%;
+}
+
+#fil {
+  width: 60%;
+  padding: 5px;
+  margin-left: 5px;
+  margin-top: 5px;
+}
+
+.panel-bg {
+  width: 87.5rem;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+}
+
+.Bookstore-CMS {
+  width: 15rem;
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
```

Comment:
kindly make them dry by creating class to implement them wherever they have the same style rules. and also check for all others its applicable